dashboard/generate.py

```diff
@@ -354,18 +346,27 @@ def run(self):
 
 def main():
     parser = argparse.ArgumentParser()
-    parser.add_argument('-d', '--debug', action='store_true')
-    parser.add_argument('-f', '--file', type = str, default = './config/utgaard.csv')
-    parser.add_argument('-r', '--refresh', type = int, default = 5)
-    parser.add_argument('-t', '--test', action='store_true')
-    parser.add_argument('-o', '--out', type = str, default = '.')
+    parser.add_argument("-d", "--debug", action="store_true")
+    parser.add_argument("-f", "--file", type=str, default="./config/utgaard.csv")
+    parser.add_argument("-r", "--refresh", type=int, default=5)
+    parser.add_argument("-t", "--test", action="store_true")
+    parser.add_argument("-o", "--out", type=str, default=".")
     args = parser.parse_args()
-
+    # edit refresh into index rather than svg, in case of svg issues
+    # file already in place from start-up script
+    with open(f"{args.out}/index.html", "r") as f:
+        newlines = []
+        for line in f.readlines():
+            newlines.append(line.replace("###refresh###", f"{args.refresh * 1000}"))
+    with open(f"{args.out}/index.html", "w") as f:
+        for line in newlines:
+            f.write(line)
```

Comment:
The webserver is running when this script starts or restarting, so to be nice, maybe you should buffer the lines in one variable and than use the sync_write_fs function I created.